app/transformations/UserAnswersToETMP/API1830.scala

```diff
@@ -41,6 +50,18 @@ object API1830 extends Transformer {
         (pathPaymentDetails \ Symbol("monetaryAmount")).json.copyFrom((__ \ Symbol("totalPensionAmounts")).json.pick)).reduce
   }
 
+  private def readsIndividualMemberDetailsEvent6(eventType: EventType): Reads[JsObject] = {
+    (
+      (pathIndividualMemberDetails \ Symbol("firstName")).json.copyFrom((__ \ Symbol("membersDetails") \ Symbol("firstName")).json.pick) and
+        (pathIndividualMemberDetails \ Symbol("lastName")).json.copyFrom((__ \ Symbol("membersDetails") \ Symbol("lastName")).json.pick) and
+        (pathIndividualMemberDetails \ Symbol("nino")).json.copyFrom((__ \ Symbol("membersDetails") \ Symbol("nino")).json.pick) and
+        (__ \ Symbol("eventType")).json.put(JsString(s"Event${eventType}")) and
+        (pathPaymentDetails \ Symbol("amountCrystalised")).json.copyFrom((pathAmountCrystallisedAndDateDetails \ Symbol("amountCrystallised")).json.pick) and
+        (pathPaymentDetails \ Symbol("typeOfProtection")).json.copyFrom((__ \ Symbol("typeOfProtection")).json.pick) and
```

Comment:
Is the format of the type of protection being saved in userAnswers the same format as whats in the API doc? I.e. isn't "enhancedLifetimeAllowance" to be transformed into "Enhanced life time allowance" when sent to ETMP?

Reply:
Amended in this commit: https://github.com/hmrc/pension-scheme-event-reporting/pull/61/commits/4371ae416fd2fe170bbf7f258a2e3720a7c3f6c8